doc/source/whatsnew/v0.20.3.txt

```diff
@@ -39,7 +39,7 @@ Bug Fixes
 - Fixed issue with dataframe scatter plot for categorical data that reports incorrect column key not found when categorical data is used for plotting (:issue:`16199`)
 - Fixed a pytest marker failing downstream packages' tests suites (:issue:`16680`)
 - Fixed compat with loading a ``DataFrame`` with a ``PeriodIndex``, from a ``format='fixed'`` HDFStore, in Python 3, that was written in Python 2 (:issue:`16781`)
-
+- Fixed bug in ``pandas.core.window_flex_binary_moment()`` when a multi-index DataFrame is passed in as ``arg1`` (:issue:`16789`)
```

Comment:
`window_flex_binary_moment` is an internal method, so the user reading these notes doesn't need to care where exactly the bug is. Better to just say something like

    Fixed bug where computing the rolling covariance of a MultiIndexed ``DataFrame`` improperly raised a ``ValueError`` (:issue:`16789`)